Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ruby:3.2-bullseye
+RUN apt-get update && apt-get install -y nodejs npm 
+
+ 
+# Default directory
+ENV INSTALL_PATH /opt/app
+RUN mkdir -p $INSTALL_PATH
+
+# Install rails
+
+#RUN chown -R user:user /opt/app
+WORKDIR /opt/app
+
+COPY . .
+
+RUN bundle install
+RUN npm install
+
+# Expose port 8888
+EXPOSE 8888
```

Comment:
pebble: is there a reason we aren't using the default port 3000?

Reply:
@tabroughton Initially, we had set it to port 3000, but encountered issues accessing the docker container locally during our tests, which led us to switch to port 8888. Interestingly, it eventually worked on port 3000, but I opted to keep it at 8888 at that moment. If there's a preference or a specific reason to revert back to port 3000, I'm more than willing to make that change.

Reply:
This has since been changed back to 3000